torrt/bots/telegram_bot.py

```diff
@@ -0,0 +1,142 @@
+import logging
+
+from torrt.base_bot import BaseBot
+from torrt.toolbox import add_torrent_from_url
+from torrt.utils import TorrtConfig, BotClassesRegistry, get_torrent_from_url, RPCObjectsRegistry
+
+LOGGER = logging.getLogger(__name__)
+
+
+START, ASKING_URL, URL, PATH = 1, 2, 3, 4
+
+
+class TelegramBot(BaseBot):
+
+    alias = 'telegram'
+    url = 'https://api.telegram.org/bot'
+
+    def __init__(self, token, allowed_users=None):
+        """
+        :param token: str - Telegram's bot token
+        :param allowed_users: str - comma-joined list of users allowed to add new torrents.
+        """
+
+        self.token = token
+        allowed_users = allowed_users or ''
+        self.handler_kwargs = {}
+        try:
+            from telegram.ext import Updater, Filters
```

Comment:
Let's make package import at the top level, like

```
try:
    import telegram
except ImportError:
    telegram = None
```

Than we check for ``None`` in funcs.